jaxb-ri/codemodel/codemodel/pom.xml

```diff
@@ -16,7 +16,7 @@
     <parent>
         <groupId>com.sun.xml.bind.mvn</groupId>
         <artifactId>jaxb-codemodel-parent</artifactId>
-        <version>2.4.0-SNAPSHOT</version>
+        <version>2.3.2-SNAPSHOT</version>
```

Comment:
version should be changed in a branch, not in master

Reply:
right